Gemfile

```diff
@@ -170,6 +170,7 @@ group :development, :test do
   gem 'rack-vcr'
   gem 'rainbow' # Used to colorize output for rake tasks
   gem 'rspec-instrumentation-matcher'
+  gem 'rspec-its'
```

Comment:
I _could_ split this addition into its own PR, but that feels like busy work TBH.